### PR TITLE
os/bluestore/bluestore-tool: minor fixes around migrate

### PR DESCRIPTION
**Pull request by ifed01**
Signed-off-by: Igor Fedotov <ifedotov@suse.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by ifed01** [comment]
@aclamk - a couple more commits, please review

File: src/os/bluestore/bluestore_tool.cc [hunk 1]
```diff
@@ -691,8 +691,14 @@ int main(int argc, char **argv)
     for (auto& s :  devs_source) {
       auto i = cur_devs_map.find(s);
       if (i != cur_devs_map.end()) {
-	src_devs.emplace(*i);
-	src_dev_ids.emplace(i->second);
+        if (s == dev_target) {
```
Comment (iliul) [inline src/os/bluestore/bluestore_tool.cc]: 👍

**Comment by tchaikov** [comment]
@aclamk ping?

**Comment by tchaikov** [comment]
```
2019-07-04T14:24:14.893 INFO:tasks.workunit.client.0.smithi177.stderr:/home/ubuntu/cephtest/clone.client.0/qa/standalone/osd/osd-bluefs-volume-ops.sh:92: TEST_bluestore:  ceph-blues
tore-tool --path td/osd-bluefs-volume-ops/0 --devs-source td/osd-bluefs-volume-ops/0/block.wal --dev-target td/osd-bluefs-volume-ops/0/block.db --command bluefs-bdev-migrate
2019-07-04T14:24:14.914 INFO:tasks.workunit.client.0.smithi177.stdout:inferring bluefs devices from bluestore path
2019-07-04T14:24:15.702 INFO:tasks.workunit.client.0.smithi177.stdout: device removed:0 td/osd-bluefs-volume-ops/0/block.wal
2019-07-04T14:24:15.708 INFO:tasks.workunit.client.0.smithi177.stderr:/home/ubuntu/cephtest/clone.client.0/qa/standalone/osd/osd-bluefs-volume-ops.sh:95: TEST_bluestore:  return 1
2019-07-04T14:24:15.708 INFO:tasks.workunit.client.0.smithi177.stderr:/home/ubuntu/cephtest/clone.client.0/qa/standalone/osd/osd-bluefs-volume-ops.sh:26: run:  return 1
```

http://pulpito.ceph.com/kchai-2019-07-04_12:44:28-rados-wip-kefu-testing-2019-07-04-1747-distro-basic-smithi/4091085

**Comment by ifed01** [comment]
@tchaikov - fixed, please re-run

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2019-07-04_12:44:28-rados-wip-kefu-testing-2019-07-04-1747-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2019-07-05_01:50:15-rados-wip-kefu-testing-2019-07-05-0137-distro-basic-smithi/

**Pull request merged**